x-pack/plugin/sql/sql-action/src/main/java/org/elasticsearch/xpack/sql/action/SqlQueryResponse.java

```diff
@@ -157,7 +158,7 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
             builder.endArray();
 
             if (cursor.equals("") == false) {
-                builder.field(SqlQueryRequest.CURSOR.getPreferredName(), cursor);
```

Comment:
Do we still need the `CURSOR` and `FILTER` in `org.elasticsearch.xpack.sql.action.SqlQueryRequest`?